oojs/application.js

```diff
@@ -1,13 +1,25 @@
 $(document).ready(function() {
```

Comment:
You only need to register the listeners inside of the `$(document).ready()` function. Let the other code be read in immediately and only register the callbacks when the document has fully rendered.